File: src/it/java/quebec/virtualite/backend_integration/CucumberIT.java
package quebec.virtualite.backend_integration;

import org.junit.platform.suite.api.ConfigurationParameter;
import org.junit.platform.suite.api.IncludeEngines;
import org.junit.platform.suite.api.SelectDirectories;
import org.junit.platform.suite.api.Suite;

import static io.cucumber.core.options.Constants.FEATURES_PROPERTY_NAME;
import static io.cucumber.core.options.Constants.PLUGIN_PROPERTY_NAME;

@Suite
@IncludeEngines("cucumber")
@SelectDirectories(".")
@ConfigurationParameter(key = FEATURES_PROPERTY_NAME, value = "src/features")
@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty, html:target/cucumber-reports/last-run.html")
public class CucumberIT
{
}

File: src/it/kotlin/quebec/virtualite/backend_integration/CucumberIT.kt
package quebec.virtualite.backend_integration

import io.cucumber.core.options.Constants.FEATURES_PROPERTY_NAME
import io.cucumber.core.options.Constants.PLUGIN_PROPERTY_NAME
import org.junit.platform.suite.api.ConfigurationParameter
import org.junit.platform.suite.api.IncludeEngines
import org.junit.platform.suite.api.SelectDirectories
import org.junit.platform.suite.api.Suite

@Suite
@IncludeEngines("cucumber")
@SelectDirectories(".")
@ConfigurationParameter(key = FEATURES_PROPERTY_NAME, value = "src/features")
@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty, html:target/cucumber-reports/last-run.html")
class CucumberIT
